themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties

```
@@ -35,26 +35,10 @@ requiredAction.UPDATE_PROFILE=Aktualizace profilu
 requiredAction.VERIFY_EMAIL=Ověření e-mailu
 
 # units for link expiration timeout formatting
-linkExpirationFormatter.timePeriodUnit.seconds=sekund
-linkExpirationFormatter.timePeriodUnit.seconds.1=sekunda
-linkExpirationFormatter.timePeriodUnit.seconds.2=sekundy
-linkExpirationFormatter.timePeriodUnit.seconds.3=sekundy
-linkExpirationFormatter.timePeriodUnit.seconds.4=sekundy
-linkExpirationFormatter.timePeriodUnit.minutes=minut
-linkExpirationFormatter.timePeriodUnit.minutes.1=minuta
-linkExpirationFormatter.timePeriodUnit.minutes.2=minuty
-linkExpirationFormatter.timePeriodUnit.minutes.3=minuty
-linkExpirationFormatter.timePeriodUnit.minutes.4=minuty
-linkExpirationFormatter.timePeriodUnit.hours=hodin
-linkExpirationFormatter.timePeriodUnit.hours.1=hodina
-linkExpirationFormatter.timePeriodUnit.hours.2=hodiny
-linkExpirationFormatter.timePeriodUnit.hours.3=hodiny
-linkExpirationFormatter.timePeriodUnit.hours.4=hodiny
-linkExpirationFormatter.timePeriodUnit.days=dní
-linkExpirationFormatter.timePeriodUnit.days.1=den
-linkExpirationFormatter.timePeriodUnit.days.2=dny
-linkExpirationFormatter.timePeriodUnit.days.3=dny
-linkExpirationFormatter.timePeriodUnit.days.4=dny
+linkExpirationFormatter.timePeriodUnit.seconds={0,choice,0#sekund|1#sekunda|2#sekundy|2<sekund}
```

In Czech, the grammatical cases differ with the number up to 5:

- 1 sekunda
- 2 sekundy
- 3 sekundy
- 4 sekundy
- 5 sekund

The current state leads to incorrect "3 sekund" though.

```suggestion
linkExpirationFormatter.timePeriodUnit.seconds={0,choice,0#sekund|1#sekunda|2#sekundy|3#sekundy|4#sekundy|4<sekund}
```

This applies to other locales as well: `ar`, `pl`, `sk`, and to the other time units too.

---

@hmlnarik - thanks for spotting this and sorry for breaking these languages.

I've pushed the necessary changes. The shorter version is to change it to `<2` to `<4`, see below for the output.

Please give it another review when you have the time. Thanks!

```
jshell> java.text.MessageFormat.format("{0,choice,0#sekund|1#sekunda|2#sekundy|4<sekund}",0)
$14 ==> "sekund"

jshell> java.text.MessageFormat.format("{0,choice,0#sekund|1#sekunda|2#sekundy|4<sekund}",1)
$15 ==> "sekunda"

jshell> java.text.MessageFormat.format("{0,choice,0#sekund|1#sekunda|2#sekundy|4<sekund}",2)
$16 ==> "sekundy"

jshell> java.text.MessageFormat.format("{0,choice,0#sekund|1#sekunda|2#sekundy|4<sekund}",3)
$17 ==> "sekundy"

jshell> java.text.MessageFormat.format("{0,choice,0#sekund|1#sekunda|2#sekundy|4<sekund}",4)
$18 ==> "sekundy"

jshell> java.text.MessageFormat.format("{0,choice,0#sekund|1#sekunda|2#sekundy|4<sekund}",5)
$19 ==> "sekund"
```